<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="docker_plugin_finalize" skipped="0" tests="1" time="4.98" timestamp="2022-04-07T17:08:59">
	<testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<failure message="BLOCKED: sudo&lt;docker_pull&gt; returned non-zero exit code 1" type="BLOCKED"/>
		<system-out>17:08:54 INFO  ==============================================================
17:08:54 INFO  Id:    docker_plugin_finalize
17:08:54 INFO  Dir:   plugin_docker/
17:08:54 INFO  Title: Validate docker plugin finalize command
17:08:54 DEBUG Execution order hint: 0.0
17:08:54 INFO  ==============================================================
17:08:54 DEBUG Process parameters for sudo&lt;docker_images&gt;
  command line : /usr/bin/sudo docker images
    arg #1     : docker
    arg #2     : images
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
17:08:54 DEBUG Waiting up to 600 secs for process sudo&lt;docker_images&gt; (pid 8748)
17:08:56 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
17:08:56 DEBUG Performing regex contains=False grep on file: /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out
17:08:56 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_images.out ... passed
17:08:56 DEBUG Process parameters for sudo&lt;docker_pull&gt;
  command line : /usr/bin/sudo docker pull registry
    arg #1     : docker
    arg #2     : pull
    arg #3     : registry
  working dir  : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux
  stdout       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_pull.out
  stderr       : /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_pull.err
  environment  : LD_LIBRARY_PATH=/usr/lib
  environment  : PATH=/bin:/usr/bin:/usr/sbin:/usr/local/bin
                   #1  /bin
                   #2  /usr/bin
                   #3  /usr/sbin
                   #4  /usr/local/bin
17:08:56 DEBUG Waiting up to 600 secs for process sudo&lt;docker_pull&gt; (pid 8758)
17:08:59 WARN  Executed sudo&lt;docker_pull&gt;, exit status 1
17:08:59 DEBUG Opening file using encoding=UTF-8: /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_finalize/Output/linux/docker_pull.err
17:08:59 INFO  Contents of docker_pull.err: 
  Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  -----

17:08:59 WARN  sudo&lt;docker_pull&gt; returned non-zero exit code 1 ... blocked 
17:08:59 WARN  Aborted test due to BLOCKED outcome
17:08:59 DEBUG --- test cleanup
17:08:59 INFO  
17:08:59 INFO  cleanup:
17:08:59 DEBUG Running registered cleanup function: &lt;bound method DockerPlugin.cleanup_containers of &lt;DockerPluginFinalizeTest object at 0x760f1bf0&gt;&gt;
17:08:59 DEBUG Running registered cleanup function: &lt;bound method DockerPlugin.cleanup_images of &lt;DockerPluginFinalizeTest object at 0x760f1bf0&gt;&gt;
17:08:59 DEBUG ProcessUser cleanup function done.
17:08:59 INFO  
17:08:59 INFO  Test duration: 4.98 secs
17:08:59 INFO  Test final outcome:  BLOCKED
17:08:59 INFO  Test outcome reason: sudo&lt;docker_pull&gt; returned non-zero exit code 1
17:08:59 INFO  
</system-out>
	</testcase>
</testsuite>
